opdracht5.js

```diff
@@ -0,0 +1,304 @@
+            speech: false,
+            hdr: false,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
+    },
+    {
+        type: '32WL1A63DG',
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
+    },
+];
+
+// #### Opdracht 5 - Functies
+
+// * **Opdracht 5a:** Zorg ervoor dat er een string wordt gegenereerd voor de naam van een tv. Maak een functie die één
+// enkel tv-object (zoals `inventory[0]` of `inventory[6]`) verwacht en de naam op de volgende manier
+// samenvoegt: `[merk] [type] - [naam]` zoals `Philips 43PUS6504/12 - 4K TV` of `NIKKEI NH3216SMART - HD smart TV`. Test
+// of jouw functie ook werkt wanneer er een ander tv object wordt meegegeven.
+
+
+function printTV (object) {
+    const printArrayTv = object.map((array) => {
+        let collection = array.brand +" " + array.type
+        let collection1 = " - " + array.name
+        return collection + collection1
+    })
+    return printArrayTv
+}
+
+console.log(printTV(inventory)[0])
+
+// * **Opdracht 5b:** Zorg ervoor dat de prijs van een tv netjes geformat wordt. Maak een functie die één tv-prijs als
+// parameter verwacht (zoals `379`) en daar de volgende string van maakt: `€379,-`. Test of jouw functie ook werkt
+// wanneer er een andere tv-prijs wordt meegegeven.
+
+function arrayPriceTv (input) {
+    const tvPrice = input.map((arr) => {
+        return arr.price + ",-"
+    })
+    return tvPrice
+}
+
+console.log(arrayPriceTv(inventory)[0])
+
+
+// * **Opdracht 5c:** Zorg ervoor dat er een string wordt gegenereerd voor alle beschikbare schermgroottes van één tv, in
+// zowel _inches_ als _cm_. Doe dit door een functie te schrijven die één screen-sizes array verwacht (
+//     zoals `inventory[0].availableSizes`) en de groottes op de volgende manier
+// samenvoegt: `[schermgrootte] inches ([schermgrootte omgerekend]cm) | [schermgrootte] inches ([schermgrootte omgerekend]cm)`
+// etc. Als een tv maar één schermgrootte heeft (`[32]`) wordt de output `32 inch (81 cm)`. Wanneer een tv vier
+// schermgroottes heeft (`[43, 50, 55, 58]`) wordt de output `43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)`.
+//     Test of jouw functie werkt in alle mogelijke gevallen.
+
+
+function testArray (arrayInvent) {
+    let inchToCm = ""
+    for (let i = 0; i < arrayInvent.length ; i++) {
+        const tvSizeInch = arrayInvent[i]
+        const tvSizeCm = tvSizeInch * 2.54
+        inchToCm = inchToCm + `${tvSizeInch} Inch | ${tvSizeCm}CM | `
+    } return inchToCm
+}
+console.log(testArray(inventory[0].availableSizes))
+
```

Comment:
Deze heb jij veel beter gemaakt dan ik :-) dus hier kan ik geen verbetering in aangeven :-D